types/graphql/type/definition.d.ts

```diff
@@ -262,9 +262,9 @@ export type GraphQLIsTypeOfFn<TSource, TContext> = (
     info: GraphQLResolveInfo
 ) => boolean | Promise<boolean>;
 
-export type GraphQLFieldResolver<TSource, TContext> = (
+export type GraphQLFieldResolver<TSource, TContext, TArgs = { [argName: string]: any }> = (
```

Comment:
@tgriesser In `graphql-js` it described as following 
```js
export type GraphQLFieldResolver<TSource, TContext> = (
```
https://github.com/graphql/graphql-js/blob/master/src/type/definition.js#L717

I strongly believe that TS typings should be in sync with official Flow typings.
Please consider contributing your changes to `graphql-js` first.

Reply:
Ah cool will do

Reply:
@IvanGoncharov Related PR: https://github.com/graphql/graphql-js/pull/1197